doc.go

```diff
@@ -0,0 +1,218 @@
+/*
+Package fx is the UberFx Service Framework.
+
+[][doc]
+[][cov]
+[]ci (https://travis-ci.org/uber-go/fx)
```

Comment:
I'm going to have to patch the upstream markdown parser, which doesn't seem to understand the syntax we're using and thinks these are "normal" text.